modules/@apostrophecms/doc-type/ui/apos/components/AposDocEditor.vue

```diff
@@ -342,6 +351,9 @@ export default {
         pref = null;
       }
       return pref;
+    },
+    showLocalePicker() {
+      return this.moduleOptions.localized !== false && !this.modalData.hasContextLocale;
```

Comment:
If the module isn't localized or we are already in a localized version of a doc we do not show the locale picker.